diff --git a/node_modules/@pqina/filepond-plugin-image-editor/dist/FilePondPluginImageEditor.d.ts b/node_modules/@pqina/filepond-plugin-image-editor/dist/FilePondPluginImageEditor.d.ts
index 77e7ebd..336c7fa 100644
--- a/node_modules/@pqina/filepond-plugin-image-editor/dist/FilePondPluginImageEditor.d.ts
+++ b/node_modules/@pqina/filepond-plugin-image-editor/dist/FilePondPluginImageEditor.d.ts
@@ -1,6 +1,6 @@
 // @ts-ignore
-import { FilePondOptions } from 'filepond';
-
+import 'filepond';
+export default FilePondPluginImageEditor
 declare module 'filepond' {
     export interface FilePondOptions {
         /** Enable or disable image editor */
